test_ffi/tests/test.js

```diff
@@ -19,7 +19,11 @@ try {
 }
 
 const dylib = Deno.dlopen(libPath, {
-  "print_something": { parameters: [], result: "void" },
+  "printSomething": {
+    name: "print_something",
+    parameters: [],
+    result: "void",
+  },
```

Comment:
Would be great to have a test for multiple overloads of a method

Reply:
Cannot use variadic arguments in stable rust. I also tried doing it in C but cannot re-export extern functions as symbols either...

Reply:
@bartlomieju can I instead add test case for multiple overloads that don't have different arguments [1] but one is sync and other one is async (using the `nonblocking` property)? It would be useful for making, for instance, an async variant of a FFI function alongside blocking one (such as `read` & `readSync`)

[1] which needs variadic arguments that cannot be used in stable Rust.

Reply:
Sure, sounds good

Reply:
I added that but CI is failing on unrelated things...

Reply:
@DjDeveloperr It is probably failing because this PR updates the `std` submodule too, is that intentional?

Reply:
No it's not intentional, seems like it updated itself when I pulled in main and resolved merge conflicts.